### PR TITLE
Adding imgrush support

### PR DESCRIPTION
**Pull request by imgrush**

### CODE REVIEW

File: Chrome/manifest.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
 				"modules/keyboardNav.js",
 				"modules/about.js",
 				"modules/commandLine.js",
+				"modules/hosts/imgrush.js",
```
Comment (allthefoxes) [inline Chrome/manifest.json]: Can this get moved down to the other /hosts/? Same for all other manifests

**Comment by allthefoxes** [comment]
Make sure to change mediacrush to imgrush in `/lib/modules/hosts/hosts.json' and uploader needs to be turned back on for this as well

**Comment by allthefoxes** [comment]
Can't get it working, seems to still be calling on mediacrush somewhere

    Uncaught ReferenceError: Imgrush is not defined - imgrush.js:3 (anonymous function)
    mediacrush.js:323 OPTIONS https://mediacru.sh/api/upload/file net::ERR_CONNECTION_TIMED_OUT - mediacrush.js:323          window.MediaCrush.self.upload
     mediacrush.js:32 
     modules.uploader.hosts.mediacrush.handleFileupload.js:93 
    handleupload.js:96 (anonymous function)

**Comment by allthefoxes** [comment]
@andytuba how hardcoded is mediacrush in this dang thing?

**Comment by imgrush** [comment]
@allthefoxes i basically run ```gulp add-module --file hosts/imgrush.js``` and it added the hosts.json and all other files. @andytuba  is there something i'm missing?

**Comment by andytuba** [comment]
@imgrush sorry i've been neglecting this!  I'm allotting Sunday to this, hope I can sort it all out by lunch.

**Comment by imgrush** [comment]
@andytuba no worries, let me how I can help

**Pull request closed without merging**